src/index.html

```diff
@@ -22,6 +22,440 @@
+          alt="moyo logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="product-cards"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="img product"
+          class="product-cards__image"
+        />
+
+        <div class="product-cards__info">
+          <h2 class="product-cards__info-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="product-cards__info-product-code">Product code: 195434</p>
+        </div>
+
+        <div class="product-cards__raiting">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <span class="product-cards__raiting-rewiews">Reviews: 5</span>
+        </div>
+
+        <div class="product-cards__prise">
+          <span class="product-cards__prise-title">Price:</span>
+          <span class="product-cards__prise-value">$2,199</span>
+        </div>
+
+        <a
+          href="#"
+          class="product-cards__button-link"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div
```

Comment:
```suggestion
      </div>

      <div
```
add blank line between empty cards to make it easier to read